docs/resources/vault_secrets_dynamic_secret.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
One of the downside of TF resources with a high level of variability is that the doc grows over time instead of being well scoped to each dynamic/rotating secret types.